nebari/deploy.py

```diff
@@ -267,31 +267,31 @@ def deploy_configuration(
     skip_remote_state_provision,
 ):
     if config.get("prevent_deploy", False):
-        # Note if we used the Pydantic model properly, we might get that qhub_config.prevent_deploy always exists but defaults to False
+        # Note if we used the Pydantic model properly, we might get that nebari_config.prevent_deploy always exists but defaults to False
         raise ValueError(
             textwrap.dedent(
                 """
-        Deployment prevented due to the prevent_deploy setting in your qhub-config.yaml file.
-        You could remove that field to deploy your QHub, but please do NOT do so without fully understanding why that value was set in the first place.
+        Deployment prevented due to the prevent_deploy setting in your nebari-config.yaml file.
+        You could remove that field to deploy your Nebari, but please do NOT do so without fully understanding why that value was set in the first place.
 
-        It may have been set during an upgrade of your qhub-config.yaml file because we do not believe it is safe to redeploy the new
-        version of QHub without having a full backup of your system ready to restore. It may be known that an in-situ upgrade is impossible
-        and that redeployment will tear down your existing infrastructure before creating an entirely new QHub without your old data.
+        It may have been set during an upgrade of your nebari-config.yaml file because we do not believe it is safe to redeploy the new
+        version of Nebari without having a full backup of your system ready to restore. It may be known that an in-situ upgrade is impossible
+        and that redeployment will tear down your existing infrastructure before creating an entirely new Nebari without your old data.
 
-        PLEASE get in touch with Quansight at https://github.com/Quansight/qhub for assistance in proceeding.
+        PLEASE get in touch with Quansight at https://github.com/Quansight/nebari for assistance in proceeding.
```

Comment:
This should be the nebari dev team and point to the GH discussions

Reply:
Done.